src/Label/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Label'
```

Comment:
This should fix the build errors
```suggestion
import Label from './Label'
export type {LabelProps} from './Label'
export default Label
```